static/js/containers/VideoEmbedPage.js

```diff
@@ -27,28 +28,71 @@ class VideoEmbedPage extends React.Component<*, void> {
   }
 
   render() {
```

Comment:
this is kind of a style thing, but I prefer to have `render` be the last method declared on a React component - would you mind terribly moving it to the bottom of the class? :smile: